sspl_test/functional_tests/alerts/os/test_systemd_service_actuator.py

```diff
@@ -0,0 +1,121 @@
+    action = "status"
+    service_name = "rsyslog.service"
+
+    def init(self):
+        pass
+
+    def request(self):
+        return service_actuator_request(self.service_name, self.action)
+
+    def filter(self, msg):
+        return actuator_response_filter(msg, self.resource_type, self.service_name)
+
+    def response(self, msg):
+        service_actuator_msg = msg.get("actuator_response_type")
+
+        assert service_actuator_msg is not None
+        assert service_actuator_msg.get("alert_type") == "UPDATE"
+        assert service_actuator_msg.get("severity") is not None
+        assert service_actuator_msg.get("host_id") is not None
+        assert service_actuator_msg.get("info") is not None
+
+        info = service_actuator_msg.get("info")
+        assert info.get("site_id") is not None
+        assert info.get("node_id") is not None
+        assert info.get("cluster_id") is not None
+        assert info.get("rack_id") is not None
+        assert info.get("resource_type") == self.resource_type
+        assert info.get("event_time") is not None
+        assert info.get("resource_id") == self.service_name
+        assert service_actuator_msg.get("specific_info") is not None
+
+
+class SystemdServiceInvalidRequestTest(TestCaseBase):
+    resource_type = "node:sw:os:service"
+    action = "start"
+    service_name = "temp_dummy.service"
+
+    def init(self):
+        pass
+
+    def request(self):
+        return service_actuator_request(self.service_name, self.action)
+
+    def filter(self, msg):
+        return actuator_response_filter(msg, self.resource_type, self.service_name)
+
+    def response(self, msg):
+        service_actuator_msg = msg.get("actuator_response_type")
+
+        assert service_actuator_msg is not None
+        assert service_actuator_msg.get("alert_type") == "UPDATE"
+        assert service_actuator_msg.get("severity") is not None
+        assert service_actuator_msg.get("host_id") is not None
+        assert service_actuator_msg.get("info") is not None
+
+        info = service_actuator_msg.get("info")
+        assert info.get("site_id") is not None
+        assert info.get("node_id") is not None
+        assert info.get("cluster_id") is not None
+        assert info.get("rack_id") is not None
+        assert info.get("resource_type") == self.resource_type
+        assert info.get("event_time") is not None
+        assert info.get("resource_id") == self.service_name
+
+        assert service_actuator_msg.get("specific_info") is not None
+        specific_info = service_actuator_msg.get("specific_info")
+        assert specific_info[0].get("error_msg") is not None
+
+
+def service_actuator_request(service_name, action):
+    egressMsg = {
+        "title": "SSPL-LL Actuator Request",
+        "description": "Seagate Storage Platform Library - Actuator Request",
+        "username": "sspl-ll",
+        "expires": 3600,
+        "signature": "None",
+        "time": "2020-03-06 04:08:04.071170",
```

Comment:
time atleast lets not hardcode, use current time, format & use it. again future PR